app/data_client_test.go

```diff
@@ -363,6 +385,29 @@ func TestDataClient(t *testing.T) {
 		test.That(t, resp, test.ShouldResemble, latestTabularData)
 	})
 
+	t.Run("ExportTabularData", func(t *testing.T) {
+		mockStream := &inject.DataServiceExportTabularDataClient{
+			RecvFunc: func() (*pb.ExportTabularDataResponse, error) {
+				return exportTabularResponse, nil
+			},
+		}
+		grpcClient.ExportTabularDataFunc = func(ctx context.Context, in *pb.ExportTabularDataRequest,
+			opts ...grpc.CallOption,
+		) (pb.DataService_ExportTabularDataClient, error) {
+			test.That(t, in.PartId, test.ShouldEqual, partID)
+			test.That(t, in.ResourceName, test.ShouldEqual, componentName)
+			test.That(t, in.ResourceSubtype, test.ShouldEqual, componentType)
+			test.That(t, in.MethodName, test.ShouldEqual, method)
+			test.That(t, in.Interval, test.ShouldResemble, captureIntervalToProto(captureInterval))
+			return mockStream, nil
```

Comment:
@purplenicole730 One thing that feels a little weird to me is that I have to use the proto response to satisfy all the interfaces for the inject and as a result I'm expecting a proto message instead of the nice golang struct. Do you have any thoughts on that/is what I'm sayin gmaking sense.

Reply:
I'm not really sure what you mean. Our aim to avoid proto types is to avoid giving the users proto types. If the inject uses the proto type, that's fine since the inject function is not what the users will be using.

Reply:
Ah ok cool, you answered my question. Thanks Nicole.